spec/test_cbw_api.py

```diff
@@ -194,3 +196,21 @@ def test_update_remote_access():
             response = client.update_remote_access('4', info)
 
             assert response is False
+
+    @staticmethod
+    def test_export_remotes_accesses_files_xlsx():
```

Comment:
Same as the other PR #35, this is in the wrong file.

Reply:
I'm waiting for my PR # 35 to be valid for not doing double work